ui/src/style/variables.less

```diff
@@ -33,7 +33,7 @@
 @highlight-border-color: #88f;
 @highlight-hover-opacity: 0.2;
 @highlight-hitbox-padding: .1em;
-@highlight-annotation-color: @s2-influential-citation-orange;
+@highlight-matching-entity-color: rgba(70, 130, 180, 0.3);
```

Comment:
I think it might be best to be consistent and use hex codes for all color variables - you can convert it using something like this website (https://www.rgbtohex.net/)! :)